ENH: Adding keepdims to np.argmin,np.argmax (#19211)
numpy/numpy: doc/release/upcoming_changes/19211.new_feature.rst

``keepdims`` optional argument added to `numpy.argmin`, `numpy.argmax`
----------------------------------------------------------------------

> ``keepdims`` argument has been added to `numpy.argmin`, `numpy.argmax`.

Reviewer: ```suggestion
``keepdims`` argument is added to `numpy.argmin`, `numpy.argmax`.
```

> By default, it is `False` and hence no behaviour change should be expected
> in existing codes using `numpy.argmin`, `numpy.argmax`. If set to ``True``,

Reviewer: ```suggestion
If set to ``True``, the axes which are reduced are left in the result as dimensions with size one.
```

the axes which are reduced are left in the result as dimensions with size one.

> In simple words, the resulting array will be having same dimensions
> as the input array.

Reviewer: ```suggestion
The resulting array has the same number of dimensions and will broadcast with the
input array.
```
